### PR TITLE
ELEMENTS-1196: add workflow for rc promotion

### PR DESCRIPTION
**Pull request by nelsonsilva**

### CODE REVIEW

File: .github/workflows/promote.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        with:
+          ref: v${{ github.event.client_payload.version }}
+
+      - run: git config user.name "nuxeo-web-ui-jx-bot" && git config user.email "ui+jx-bot@nuxeo.com"
+
+      - run: echo "::set-env name=VERSION::$(echo '${{ github.event.client_payload.version }}' | sed -e 's/-rc.[0-9]*//')"
+
+      - name: Update Nuxeo Elements version to ${{ env.VERSION }}
+        run: |
+          npx lerna version $VERSION --no-push --force-publish --no-git-tag-version --yes
```
Comment (Gabez0r) [inline .github/workflows/promote.yaml:27]: We also don't have this on the RC script, but shouldn't we also be doing something like this on both:
```
          npm version $VERSION --no-git-tag-version
```
See https://github.com/nuxeo/nuxeo-elements/blob/v3.0.1-rc.2/package.json#L6.

Comment (nelsonsilva) [reply .github/workflows/promote.yaml:27]: we should yes, I'd swear I had added it but apparently only did it when aligning on next snapshot

Comment (nelsonsilva) [reply .github/workflows/promote.yaml:28]: done 👍

File: README.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
 [![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://www.webcomponents.org/element/nuxeo/nuxeo-elements)
-[![Build Status](https://qa.nuxeo.org/jenkins/buildStatus/icon?job=master/nuxeo-elements-master)](https://qa.nuxeo.org/jenkins/job/master/job/nuxeo-elements-master/)
+[![Build Status](https://jenkins.webui.dev.nuxeo.com/buildStatus/icon?job=nuxeo%2Fnuxeo-elements%2Fmaster)](https://jenkins.webui.dev.nuxeo.com/job/nuxeo/job/nuxeo-elements/job/master/)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/b6e719e30d53435e8a76230067aade3b)](https://www.codacy.com/app/Nuxeo/nuxeo-elements)
 [![Coverage Status](https://coveralls.io/repos/github/nuxeo/nuxeo-elements/badge.svg)](https://coveralls.io/github/nuxeo/nuxeo-elements)
 
 [![Browser Status](https://badges.herokuapp.com/sauce/nuxeo-elements?name=nuxeo-elements-master)](https://saucelabs.com/u/nuxeo-elements)
 
+[![Actions Panel](https://img.shields.io/badge/actionspanel-enabled-brightgreen)](https://www.actionspanel.app/app/nuxeo/nuxeo-elements)
```
Comment (Gabez0r) [inline README.md:8]: Do we still need this?

Comment (nelsonsilva) [reply README.md:8]: Nope. Good catch 👍

**Pull request merged**